apps/console/src/extensions/configs/common.tsx

```diff
@@ -36,6 +37,7 @@ export const commonConfig: CommonConfig = {
             _associatedTenants: any[]
         ): Promise<HeaderExtension[]> => Promise.resolve([]),
         getHeaderSubPanelExtensions: (): HeaderSubPanelItemInterface[] => [],
+        getUserDropdownFooterExtensions: (): ReactElement => <></>,
```

Comment:
IMO, it's better to return `null` rather than an empty fragment. WDYT?

Reply:
Fixed with 6d00a2c

Reply:
You have tested the functionality with this change, right?

Reply:
yes